t04_refactored.py

```diff
@@ -224,9 +224,43 @@ def team_18_adv():
 
 ###################################################################################
 
+
 def team_19_adv():
+
```

Comment:
missing docstring

Reply:
In your function’s docstring, add a link to your team’s Google Doc (this document) as well as the names of all partners who worked on that function. For example: